src/qt/ve_qitems_mqtt.cpp

```diff
@@ -285,6 +285,7 @@ void VeQItemMqttProducer::transitionState()
 	if (haveTransition) {
 		// transition to the appropriate state, then perform side effects
 		// associated with that state.
+		// qDebug() << "Transitioning from" << mConnectionState << " to " << transition.to;
```

Comment:
is this intentional? (it's ok if it is)

Reply:
Yes, I find myself adding this line of code every time I need to debug a problem.  So, figured I'd just leave it here, but commented out, so in future I can just easily uncomment it while debugging.